common/src/main/java/org/popcraft/chunky/integration/SquaremapIntegration.java

```diff
@@ -125,4 +128,12 @@ private Marker ellipse(Point center, double radiusX, double radiusZ) {
         points[numPoints] = Point.of(center.x(), center.z() + radiusZ);
         return Marker.polyline(points);
     }
+
+    private Optional<WorldIdentifier> getBukkitWorldIdentifier(final World world) {
+        org.bukkit.World bukkitWorld = Bukkit.getWorld(world.getUUID());
```

Comment:
I am fully aware that I am introducing bukkit code back into common which isn't ideal - the only reason this even works is because of the transitive dependency mess that exists within common, which has dependencies that adds bukkit to the classpath when it shouldn't - which is easy to fix, but forces you to fix the core problem of other platform specific code existing within common... which isn't in scope of this PR. We will address this later per discussions we had.

We will want to split the bukkit adapter stuff into the bukkit module as part of the larger platform agnostic rework and keep squaremap in common as jmp plans on making squaremap multi platform, so it wouldn't make sense to move it and then move it back again.

Reply:
Sounds good but definitely something that should get fixed in the near future.